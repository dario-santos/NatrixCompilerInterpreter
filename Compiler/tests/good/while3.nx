val x : int = 0;

while(x < 10)
{
    val y : int = 0;

    while(y < 5)
    {
        print(y + x);

        y := y + 1;
    }

    printn(x);
    
    x := x + 1;
}
