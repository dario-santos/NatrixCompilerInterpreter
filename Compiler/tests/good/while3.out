012340
123451
234562
345673
456784
567895
6789106
78910117
891011128
9101112139
